src/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes.csproj

```diff
@@ -13,8 +13,9 @@
     <Owners>Microsoft, VisualStudioExtensibility</Owners>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <PackageProjectUrl>https://github.com/Microsoft/vs-threading</PackageProjectUrl>
-    <PackageIconUrl>https://aka.ms/VsExtensibilityIcon</PackageIconUrl>
+    <PackageIcon>packageIcon.png</PackageIcon>
     <PackageReleaseNotes>In the v16.0 release, analyzers require at least C# 7.3 (Visual Studio 2017 Update 7).</PackageReleaseNotes>
+    <NoWarn>$(NoWarn),NU5128</NoWarn>
```

Comment:
I thought this was a semicolon-delimited list. It certainly is [everywhere else](https://source.dot.net/#MSBuildProperty=nowarn) I've ever seen. Can you use semicolon here?

Reply:
It's documented as a comma-separated list.

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/nowarn-compiler-option

https://github.com/microsoft/msbuild/blob/72f86ff1cb1daad46fd454c14f2dfa490e6fc49b/src/MSBuild/Microsoft.Build.CommonTypes.xsd#L1580-L1584

Reply:
Well, C# isn't the only tool that reads from that property. NuGet, FxCop (legacy perhaps), etc. also read from it. And I can't find a *single* example of using commas in all the .NET Core source code that I linked to earlier. 
Only one of the hits on [the first page of github search results](https://github.com/search?l=XML&q=nowarn&type=Code) use commas as well.

So if semicolon's work as an undocumented internal behavior, it may as well be documented now because it isn't ever going to change. So I'm more comfortable doing what 99% other code does than following documented behavior of one of many tools that read from that property.

Reply:
Resolved by using `SuppressDependenciesWhenPacking`, which I discovered in NuGet/Home#8583 and appears to be a superior solution. I did a diff of the contents of the resulting package and it's clearly an improvement.